api/src/main/java/org/openmrs/module/patientflags/metadatadeploy/handler/FlagDeployHandler.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.module.patientflags.metadatadeploy.handler;
+
+import org.openmrs.annotation.Handler;
+import org.openmrs.module.metadatadeploy.handler.AbstractObjectDeployHandler;
+import org.openmrs.module.patientflags.Flag;
+import org.openmrs.module.patientflags.api.FlagService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@Handler(supports = {Flag.class})
+
```

Comment:
Leaving the above empty line makes it look like it is not an annotation for the class